src/lwc/geFormRenderer/geFormRenderer.js

```diff
@@ -70,6 +73,9 @@ export default class GeFormRenderer extends NavigationMixin(LightningElement) {
     @api donorRecordId;
     @api donorApiName;
     @api donorRecord;
+    @api fabricatedCardholderNames;
+
+    @api loadingText;
```

Comment:
Does loadingText need to be in geFormRenderer? It looks like it's now being used in the App....

Reply:
Yep! Only the App is aware of where we are in the save flow and so we have to ferry that info into the renderer for the UI through this public `loadingText` property.

Reply:
@ptewson-sfdo